README.md

```diff
@@ -61,6 +61,8 @@ use: "file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images
 }
 ```
 
+`cssOutputPath` also should be used (together with `useRelativePath` property) to define the output path context.
```

Comment:
No, that's fine; It's fair, but I ain't finished yet. ^^
I'll come back later and then I try to write something better.